### PR TITLE
feat(actions): Add AWS Security Hub as an Action.

### PR DESCRIPTION
**Pull request by simar7**
Implements: https://github.com/aquasecurity/postee/issues/444

Signed-off-by: Simar <simar@linux.com>

### CODE REVIEW

**Comment by simar7** [comment]
ping @krol3 - could you review this if you get a chance?

**Comment by simar7** [comment]
I'm going to merge this in for now.

File: rego-filters/Trivy AWS Findings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package postee
+
+allow {
+    contains(input.Findings[0].ProductFields["Product Name"], "Trivy")
```
Comment (krol3) [inline rego-filters/Trivy AWS Findings:4]: About the names inside the rego-filters, don't be better use a pattern with lowercase and without spaces, for example: `trivy-aws-findings`.

Also maybe we could use a pattern to create the sample rego-filters, for example [inputEvent]-[sample].rego. 

Allow-Image-Name.rego -> trivy-allowImage.rego
Tracee Default Set -> tracee-default-set

Comment (simar7) [reply rego-filters/Trivy AWS Findings:4]: Maybe, today the filenames are doubled up as UI element names. We'll need to change that if we want to define a naming schema.

**Pull request merged**